Source/DafnyRuntime/DafnyRuntime.csproj

```diff
@@ -9,6 +9,7 @@
       <PackageVersion>1.1.0</PackageVersion>
       <TargetFramework>net6.0</TargetFramework>
       <OutputPath>..\..\Binaries\</OutputPath>
+      <LangVersion>7.1</LangVersion>
```

Comment:
The [default](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version) for even the ancient dotnet versions is `7.3`. Should we maybe target that?